clang/test/dpct/math_functions_test.cu

```diff
@@ -68,10 +68,10 @@ float test_sqrt(float in) { return sqrt(in); }
 // CHECK: float test_pow(float a, float b) { return pow(a, b); }
 float test_pow(float a, float b) { return pow(a, b); }
 
-// CHECK: float test_pow(float a, int b) { return sycl::pown(a, b); } 
+// CHECK: float test_pow(float a, int b) { return pow(a, b); }
```

Comment:
if --optimize-migration is speicified, consider to migrate to sycl::pown()

Reply:
It is a pure host API and its declaration is not from cuda header, so I think keep the original code is better.